src/app/component/user/components/profile/shopping-list/shopping-list.component.ts

```diff
@@ -1,19 +1,50 @@
+import { ProfileService } from './../profile-service/profile.service';
 import { Component, OnInit } from '@angular/core';
+import { ShoppingList } from '@global-user/models/shoppinglist.model';
 
 @Component({
   selector: 'app-shopping-list',
   templateUrl: './shopping-list.component.html',
   styleUrls: ['./shopping-list.component.scss']
 })
 export class ShoppingListComponent implements OnInit {
-  public shoppingList: Array<string> = ['Шоппер', 'Еко чашка', 'Бамбукова щітка'];
+  public shoppingList;
+  public profileSubscription;
 
-  constructor() { }
+  constructor(private profileService: ProfileService) { }
 
-  ngOnInit() {}
+  ngOnInit() {
+    this.getShoppingList();
+  }
+
+  public getShoppingList(): void {
+    this.profileSubscription = this.profileService.getShoppingList().subscribe(
+      (success: ShoppingList[]) => this.shoppingList = success,
```

Comment:
Please, check if everything is OK with punctuation

Reply:
done